stdlib/runtime/numberUtils.gr

```diff
@@ -980,7 +1012,8 @@ export let utoa64 = (value, radix) => {
       out
     }
   } else if (radix == 16n) {
-    let decimals = WasmI32.shrU(63n - WasmI32.wrapI64(WasmI64.clz(value)), 2n) + 1n
+    let decimals = WasmI32.shrU(63n - WasmI32.wrapI64(WasmI64.clz(value)), 2n) +
+    1n
```

Comment:
Does this wrapping seem weird? @ospencer

Reply:
The only other way to break that line is to push the stuff after the = to a new line.  The Reason formatter does that for all assignment, I've not so far